apps/web/src/app/globals.css

```diff
@@ -237,6 +237,35 @@
   }
 }
 
+/* Touch action utilities for Safari drag/pan gesture compatibility */
+@utility touch-pan-x {
+  touch-action: pan-x;
+}
+
+@utility touch-pan-y {
+  touch-action: pan-y;
+}
+
+@utility touch-pan-xy {
+  touch-action: pan-x pan-y;
+}
+
+@utility touch-none {
+  touch-action: none;
+}
+
+@utility touch-manipulation {
+  touch-action: manipulation;
+}
+
```

Comment:
you dumb?

this already exists in tailwind:

Class | Styles
touch-auto | touch-action: auto;
touch-none | touch-action: none;
touch-pan-x | touch-action: pan-x;
touch-pan-left | touch-action: pan-left;
touch-pan-right | touch-action: pan-right;
touch-pan-y | touch-action: pan-y;
touch-pan-up | touch-action: pan-up;
touch-pan-down | touch-action: pan-down;
touch-pinch-zoom | touch-action: pinch-zoom;
touch-manipulation | touch-action: manipulation;